src/Postnl.php

```diff
@@ -173,6 +173,7 @@ public function setShippingStatusClient(ShippingStatusClient $shippingStatusClie
      */
     public function generateBarcode(
         $type,
+        $eps = false,
```

Comment:
I guess it doesn't make much sense to add another function for generating barcodes when `generateBarcodebyDestination` is probably good enough for almost everyone if not everyone. So forget about that.